fog/view/enclave/api/src/lib.rs

```diff
@@ -8,7 +8,7 @@
 
 extern crate alloc;
 
-use alloc::{collections::BTreeMap, vec::Vec};
+use alloc::{collections::BTreeMap, string::String, vec::Vec};
```

Comment:
I don't think this change is necessary
```suggestion
use alloc::{collections::BTreeMap, vec::Vec};
```